tests/unit/nupic/regions/pcanode_test.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2014, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import unittest2 as unittest
+
+from nupic.regions.PyRegion import PyRegion
+from nupic.engine import *
+
+class PCANodeTest(unittest.TestCase):
+
+
+ def testPCANode(self):
+  rgen = numpy.random.RandomState(37)
+
+  inputSize = 8
+
+  net = Network()
+  sensor = net.addRegion('sensor', 'py.ImageSensor' ,
+          '{ width: %d, height: %d }' % (inputSize, inputSize))
+
+  params = """{bottomUpCount: %d,
+              SVDSampleCount: 5,
+              SVDDimCount: 2}""" % inputSize
```

Comment:
I suppose the SVDDimCount might account for that... I guess if it works then it's fine as is